ipatests/test_integration/test_idm_api.py

```diff
@@ -0,0 +1,534 @@
+    ipauserauthtype="otp")
+    cmd = api.Command["user_show"]("jsmith", all=True)["result"]
+    assert 'otp' in cmd['ipauserauthtype']
+    assert 'John Smith' in cmd['cn']
+        """
+        )
+        self.create_and_run_script(
+            "/tmp/user_add.py", user_code_script
+        )
+
+    def test_idm_user_find(self):
+        """
+        This test checks that user is displayed
+        using api.Command["user_find"]
+        """
+        user_code_script = textwrap.dedent(
+            f"""
+    {API_INIT}
+    cmd = api.Command["user_find"]("jsmith")
+    assert '1 user matched' in cmd['summary']
+        """
+        )
+        self.create_and_run_script(
+            "/tmp/user_find.py", user_code_script
+        )
+
+    def test_idm_user_mod(self):
+        """
+        This test checks that user attribute is modified
+        using api.Command["user_mod"]
+        """
+        user_code_script = textwrap.dedent(
+            f"""
+    {API_INIT}
+    cmd = api.Command["user_mod"]("jsmith",
+    mail="jsmith@example.org")["result"]
+    assert 'jsmith@example.org' in cmd['mail']
+        """
+        )
+        self.create_and_run_script(
+            "/tmp/user_mod.py", user_code_script
+        )
+
+    def test_disable_user(self):
+        """
+        This test checks that user is disabled
+        using api.Command["user_disable"]
+        """
+        user_code_script = textwrap.dedent(
+            f"""
+    {API_INIT}
+    cmd = api.Command["user_disable"]("jsmith")
+    assert 'Disabled user account "jsmith"' in cmd['summary']
+        """
+        )
+        self.create_and_run_script(
+            "/tmp/disable_user.py", user_code_script
+        )
+
+    def test_enable_user(self):
+        """
+        This test checks that user is enabled
+        using api.Command["user_enable"]
+        """
+        user_code_script = textwrap.dedent(
+            f"""
+    {API_INIT}
+    cmd = api.Command["user_enable"]("jsmith")
+    assert 'Enabled user account "jsmith"' in cmd['summary']
+        """
+        )
+        self.create_and_run_script(
+            "/tmp/enable_user.py", user_code_script
+        )
+
+    def test_create_ipa_group(self):
+        """
+        This test checks that group is created
+        using api.Command["group_add"]
+        """
+        user_code_script = textwrap.dedent(
+            f"""
+    {API_INIT}
+    cmd = api.Command["group_add"]("developers", gidnumber=500,
+    description="Developers")
+    assert 'Added group "developers"' in cmd['summary']
+        """
+        )
+        self.create_and_run_script(
+            "/tmp/create_group.py", user_code_script
+        )
+
+    def test_show_ipa_group(self):
+        """
+        This test checks that group is displayed
+        using api.Command["group_show"]
+        """
+        user_code_script = textwrap.dedent(
+            f"""
+    {API_INIT}
+    cmd = api.Command["group_show"]("developers")["result"]
```

Comment:
This line should be `cmd = api.Command["group_show"]("developers")` (remove the `["result"]` at the end of the line)